go/str_split.go

```diff
@@ -0,0 +1,38 @@
+package pehape
+
+import (
+	"errors"
+	"math"
+)
+
+var (
+	ErrInvalidSegmentLength = errors.New("the length of each segment must be greater than zero")
+)
+
+// StrSplit splits a string into an array.
+// Parameters
+// - str => string to be splitted
+// - length => (optional) only first index is used. Specifies the length of each array element. Default is 1.
+// Returns
+// - res => array of splitted string
+// - err => error if length less than 1
+func StrSplit(str string, segLength ...int) (res []string, err error) {
```

Comment:
ini dibuat `naked return` ajasih, dari pada `named return`. biar konsisten sama yg lain aja

```suggestion
func StrSplit(str string, segLength ...int) ([]string, error)
```

Reply:
naked return sama named return itu sama aja pak wkwkwk

Reply:
okee